### ISSUE TITLE
[A11y Select and Start Debugging] Screen reader does not announce the options completely from the drop-down menu

### ISSUE THREAD

**Issue by kuojianlu** [microsoft/vscode#143248]
Does this issue occur when all extensions are disabled?: Yes

- VS Code Version: 1.64.2
- OS Version: Win 10

Steps to Reproduce:

1. Open a empty folder, create .vscode/launch.json and add several launch configurations
2. Run command "Debug: Select and Start Debugging"

![image](https://user-images.githubusercontent.com/37978464/154394789-7471f7fe-4c72-4375-84a0-5ab3c4af8d6e.png)

Screen reader announces "Run Script: test" but does not announce "**contributed**". Similarly Screen reader announces "add contribution" but does not read "**configure**".

This bug occurs for NVDA and JAWS both.

**Comment by weinand** [comment]
@isidorn this is an issue with the QuickPick and not something debug related?

**Comment by isidorn** [comment]
Yeah dup of https://github.com/microsoft/vscode/issues/141856
fyi @TylerLeonhardt
